src/plugins/gui/providers/bityProvider.ts

```diff
@@ -490,17 +499,35 @@ const completeSellOrder = async (approveQuoteRes: BityApproveQuoteResponse, core
   }
 
   const spendInfo: EdgeSpendInfo = {
+    savedAction: {
+      type: 'sell',
+      orderId: id,
+      orderUri: '',
```

Comment:
This prop is optional according to your core branch, should just leave it undefined for clarity